src/functionalTest/resources/application-functional.yaml

```diff
@@ -34,7 +34,7 @@ ccd-data:
   name: ccd_data
   secret: ${IDAM_KEY_CCD_DATA_STORE:AAAAAAAAAAAAAAAB}
 
-ccd:
+remote:
   case-definition.host: ${DEFINITION_STORE_HOST:http://localhost:4451}
```

Comment:
Sorry, just thinking out loud. It my opinion this would look a lot more natural and less repetitive in this form:
```
remote-hosts:
  case-definition: ...
```
Or if there is chance/plan that those remote clients might have other properties, then something like this:
```
remote:
  hosts:
    case-definition: ...
```
And maybe, just maybe, instead of `remote`, say `hmcts-service` (or just `service`)